shared/src/lib/tabulator.ts

```diff
@@ -50,6 +50,9 @@ export default {
     const result = `<pre>${cellValue}</pre>`
     return result;
   },
+  isDateTime(dataType: string|null) {
```

Comment:
I think this is fine for now, but I think we probably need to add `bksType` to our `ColumnDefinition` interface at some point to identify stuff like this at the driver level, rather than in the GUI